### ISSUE TITLE
Smoke Tests: Enable more tests against PRs

### ISSUE THREAD

**Issue by midleman** [posit-dev/positron#4879]
Now that tests are running in parallel, the smoke tests only take about 3 minutes to complete for PRs. This means we have room to enable more tests for PRs, which would provide better test coverage and boost confidence before merging code.

Tasks:
* Figure out how much time we are comfortable with the tests taking?
* Determine which tests we'd like to enable staying within that allotment.

**Comment by midleman** [comment]
No longer relevant. Closing.
